frontend/src/pages/_app.tsx

```diff
@@ -1,14 +1,20 @@
 import { AppProps } from "next/app";
 
-import Navigation from "../components/Navigation";
+import Landing from "../components/Landing";
+
 import "../styles/global.css";
 import "../styles/globals.css";
 
+// import Navigation from "../components/Navigation";
+
 function App({ Component, pageProps }: AppProps) {
   return (
-    <Navigation>
+    <Landing {...pageProps}>
       <Component {...pageProps} />
-    </Navigation>
+    </Landing>
+    // <Navigation>
```

Comment:
Could you update the Navigation component so that it routes between home, programs, profile, notifications instead of index, program, profile, notifications? Should been changed when merging the NavBar PR, thats my bad for not catching that.

Reply:
I don't believe we have a "home" page, can you clarify what you mean by this?

Reply:
Right now, the index.tsx file contains a Home function that it exports.

Reply:
@parth4apple will resolve this in the navbar update PR